discount/src/main/java/com/kyriba/conference/discount/api/dto/DiscountUpdateParamsDto.java

```diff
@@ -11,7 +13,9 @@
  * @author Igor Lizura
  */
 @ApiModel(value = "DiscountUpdateParams", description = "Discount update parameters")
-@Getter
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
 public class DiscountUpdateParamsDto {
```

Comment:
Strange name for DTO

Reply:
I considered the variant with different verbs and different DTO's (PercentageDto in this case), but preferred one extensible DTO with a single verb. I'm ready to take the advice :)

Reply:
If you incapsulate data to update some logical part (X for future referencing) of domain entity, then DTO is usually named like DiscountXUpdateDto or DiscountXUpdateRequest (to differ api DTO from business logic DTOs).